app/templates/components/hopper-frontend.hbs

```diff
@@ -60,3 +61,50 @@
         </div>
     </div>
 </div>
+
+<div id="hopperOutletHelper"></div>
+
+
+<ol class="joyride-list" data-joyride>
+
+  <li data-text="Next" data-options="prev_button: false">
+    <h4>How to create a form</h4>
+    <p>
+        Welcome to our short tutorial, how to use this form constructor.
+    </p>
+  </li>
+
+  <li data-id="formTitle" data-options="tip_location: bottom;" data-text="Next" data-prev-text="Prev">
+    <h4>Changing the form title</h4>
+    <p>
+        To change the title double click at the current one. After you are done, just click somewhere else or press the enter button.
+    </p>
+  </li>
+
+  <li data-id="elementDrawer" data-options="tip_location: left;" data-text="Next" data-prev-text="Prev">
+    <h4>Adding a new field</h4>
+    <p>
+        If you wish to add a new field, you can click on this "drawer" to open it and show all possible fields. Select the field you want to add and click on it to do so. It will be appended at the end of your form where you can start filling it its attributes, like the Label or for example a placeholder.
+    </p>
+  </li>
+
+  <li data-id="hopperOutletHelper" data-options="tip_location: bottom;" data-text="Next" data-prev-text="Prev"      data-class="move-down" >
+    <h4>Editing an existing field</h4>
+    <p>
+        Down here you find your already added fields. To edit one, simply double click on it's label. After you are done with editing, click the check mark. If you want to delete a field, click on the cross.
```

Comment:
Please change `If you want to delete a field` to `If you want to delete the whole field`, because it's already known.